src/pages/recentDeliveries/csvExport.tsx

```diff
@@ -0,0 +1,72 @@
+
+import {buildRequestTableColumns, StaticTableColumns} from "./helpers";
+import jsdownload from "js-file-download";
+
+   export function CSVGenerate(requests) {
+        // let =requests
+        // let zarray=Object.entries(y)
+        // console.log(typeof(x))
+        // console.log(Object.entries(x))
+        // console.log(typeof(y))
+        // console.log(Object.entries(y))
+        // console.log(zarray)
+
+        const csvString=[
+
+          StaticTableColumns.map(item => item.label),
+            requests.map(req => 
+              StaticTableColumns.map(col => 
+                 req[col.dataKey!]
+                )
+            
+            )
+
+        ]
+
+        .map(e => e.join("\t")) 
+        .join("\n");
+
+        console.log(csvString)
+
+        jsdownload(csvString,"report.csv");
+        
+
```

Comment:
it's a little hard to tell what's still in use here?

Could you do some clean up here?

Reply:
still a work in progress! trying to get the CSV-string right after we introduced the StaticTableColumn component.